platform/qt_location_service.cpp

```diff
@@ -0,0 +1,172 @@
+#include "platform/qt_location_service.hpp"
+
+#include "base/logging.hpp"
+#include "base/macros.hpp"
+
+#include "std/target_os.hpp"
+
+#include <QGeoPositionInfoSource>
+
+namespace {
+  static location::GpsInfo gpsInfoFromQGeoPositionInfo(const QGeoPositionInfo& i, location::TLocationSource source)
```

Comment:
{ from the new line and usually we don't use 2-space indent after namespace:
```
namespace
{
// code starts here
static location::GpsInfo gpsInfoFromQGeoPositionInfo ...
}  // namespace
```

Reply:
Done